CHANGELOG.rst

```diff
@@ -4,6 +4,25 @@ Dellemc.Powerstore Change Logs
 
 .. contents:: Topics
 
+v3.8.0
+======
+
+Minor Changes
+-------------
+
+- Added support for PowerStore Victory Plus platform (4.1.x).
+
```

Comment:
Please remove product code name from the description, it's an internal-only name, no need to expose to the public community.

Reply:
@forrestxia , fixed, thanks